src/ol-ext/format/featurehash.js

```diff
@@ -337,6 +357,8 @@ ngeo.format.FeatureHash.encodeStyleText_ = function(textStyle, encodedStyles) {
 
 
 /**
+ * Read a logical sequence of characters and return a corresponding
+ * ol.geom.LineString.
```

Comment:
use `{@link ol.geom.LineString}` it' possible to make it working by modifying the template.
also bellow...